server/zio-http-server/src/test/scala/sttp/tapir/server/ziohttp/ZioHttpTestServerInterpreter.scala

```diff
@@ -16,10 +17,10 @@ import zio.interop.catz._
 import java.util.concurrent.atomic.AtomicInteger
 
 class ZioHttpTestServerInterpreter(nettyDeps: EventLoopGroup with ServerChannelFactory)
-    extends TestServerInterpreter[Task, ZioStreams, Http[Any, Throwable, Request, Response[Any, Throwable]]] {
+    extends TestServerInterpreter[Task, ZioStreams with WebSockets, Http[Any, Throwable, Request, Response[Any, Throwable]]] {
```

Comment:
don't we need to update the docs as well?

Reply:
updated